### PR TITLE
Alessio fix upgrade and downgrade

### LINKED ISSUE #177: ErrorException: Notice: Trying to access array offset on value of type null
Sentry Issue: [WUBTITLE-WORDPRESS-1Z](https://sentry.io/organizations/ctmobi/issues/1789834729/?referrer=github_integration)

```
ErrorException: Notice: Trying to access array offset on value of type null
  File "/wp-content/plugins/Wubtitle/includes/Dashboard/Templates/downgrade_plan_template.php", line 45, in include
    <img class="card_plan" src="<?php echo esc_url( WUBTITLE_URL . 'assets/img/' . $plans[ $current_plan ]['icon'] ); ?>">
  File "/wp-content/plugins/Wubtitle/includes/Dashboard/PaymentTemplate.php", line 49, in Wubtitle\Dashboard\PaymentTemplate::change_plan_template
    include $includes_file;
...
(1 additional frame(s) were not displayed)
```

### LINKED ISSUE #178: ErrorException: Notice: Undefined index: price_1GzNsvHSeRtuacxg6L5XhF8P
Sentry Issue: [WUBTITLE-WORDPRESS-1Y](https://sentry.io/organizations/ctmobi/issues/1789834725/?referrer=github_integration)

```
ErrorException: Notice: Undefined index: price_1GzNsvHSeRtuacxg6L5XhF8P
  File "/wp-content/plugins/Wubtitle/includes/Dashboard/Templates/downgrade_plan_template.php", line 46, in include
    <h1 class="title" > <?php echo esc_html( $plans[ $current_plan ]['name'] ); ?> </h1>
  File "/wp-content/plugins/Wubtitle/includes/Dashboard/PaymentTemplate.php", line 49, in Wubtitle\Dashboard\PaymentTemplate::change_plan_template
    include $includes_file;
...
(1 additional frame(s) were not displayed)
```

### LINKED ISSUE #179: ErrorException: Notice: Undefined index: price_1GzNu2HSeRtuacxgp0n5GGA6
Sentry Issue: [WUBTITLE-WORDPRESS-G](https://sentry.io/organizations/ctmobi/issues/1756421685/?referrer=github_integration)

```
ErrorException: Notice: Undefined index: price_1GzNu2HSeRtuacxgp0n5GGA6
  File "/wp-content/plugins/Wubtitle/includes/Dashboard/PaymentTemplate.php", line 42, in Wubtitle\Dashboard\PaymentTemplate::change_plan_template
    $wanted_plan   = $map_plans[ $plan ];
```

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:
Fix upgrade and downgrade page. 
Fix error:

- Undefined index: price_1GzNsvHSeRtuacxg6L5XhF8P

- Undefined index: price_1GzNu2HSeRtuacxgp0n5GGA6

- ErrorException: Notice: Trying to access array offset on value of type null #177

Closes #177 .
Closes #178 .
Closes #179 .

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage decreased per file
===========================
- includes/Api/ApiPricingPlan.php  -4
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5881537&bid=19250787)

File: includes/Api/ApiPricingPlan.php [hunk 1]
```diff
@@ -55,6 +58,7 @@ public function change_plan() {
 			WUBTITLE_ENDPOINT . 'stripe/customer/update',
 			array(
 				'method'  => 'POST',
+				'timeout' => 13,
```
Comment (HyprCoder) [inline includes/Api/ApiPricingPlan.php:61]: why the timeout is 13?

Comment (cpalessio96) [reply includes/Api/ApiPricingPlan.php:61]: because the lambda function has a 12 second timeout, I added a second for safety

Comment (HyprCoder) [reply includes/Api/ApiPricingPlan.php:61]: put 15 seconds

**Pull request merged**